pmm-app/tests/pages/databaseChecksPage.js

```diff
@@ -9,36 +11,40 @@ module.exports = {
     disabledSTTMessage: "Security Threat Tool is disabled. You can enable it in",
   },
   fields: {
-    dbCheckPanelSelector: "//div[@data-qa='db-check-panel']",
-    dbCheckPanelEmptySelector:"//div[@data-qa='db-check-panel-table-empty']",
-    sttEnabledDBCheckPanelSelector:"//div[@data-qa='db-check-panel-home']",
-    disabledSTTMessageSelector:"//div[@data-qa='db-check-panel-settings-link']",
-    serviceNameHeaderSelector:"//table[@data-qa='db-check-panel-table']//th[text()='Service name']",
-    detailsHeaderSelector:"//table[@data-qa='db-check-panel-table']//th[text()='Details']",
-    noOfFailedChecksHeaderSelector:"//table[@data-qa='db-check-panel-table']//th[text()='Failed Checks']",
-    serviceNameSelector:"//tbody/tr/td[1]",
-    failedChecksRowSelector:"//tbody/tr",
-    tooltipSelector: "//div[@class='ant-tooltip-inner']/div/div[1]",
-    totalFailedChecksTooltipSelector: "//div[@class='popper']//div[text()='Failed checks: ']",
-    failedChecksTooltipSelector: "//div[@class='popper']/div/div/div"
+    dbCheckPanelSelector: "$db-check-panel",
+    dbCheckPanelEmptySelector: "$db-check-panel-table-empty",
+    sttEnabledDBCheckPanelSelector: "$db-check-panel-home",
+    disabledSTTMessageSelector: "$db-check-panel-settings-link",
+    serviceNameSelector: "tbody > tr > td:first-child",
+    totalFailedChecksTooltipSelector: ".popper > div > div > div:first-of-type",
+    failedChecksTooltipSelector: ".popper > div > div > div",
```

Comment:
@atymchuk the more I see the CSS selectors, I don't agree with them, atleast I don't see any difference either in the readability of the locators, or them being tightly coupled to HTML structure as we discussed because to me 
`.popper > div > div > div` in no different then `//div[@class='popper']/div/div/div`, It is faster to prepare xpath locators over CSS for sure.

Reply:
@puneet0191 There are two reasons for this:
1/ css is native to the browser and would work in the browser, so this is something any developer would understand, which is not the case with xpath. You happen to take a very simple selector, but example like this one demonstrate the case much better: `"//span[contains(text(), 'N/A')]//ancestor::div[contains(@class,'panel-container')]//span[contains(@class,'panel-title-text')]"`. We can refer to sites like codecept.io and many others, who also recommend to prefer ccs over xpath. And they do it for the very same reason.
2/ Decoupling the selectors is a real challenge. It's hardly achievable to do it everywhere, but as we discussed, we will try to address it as much as we can by using custom locators and specific `data-` attributes. The readability there is outstanding, because we can give them semantic names.
3/ The time to make an css selector is equal or inferior to the time to make an xpath selector. The modern browsers feature excellent tooling for both of them (see attachment).
4/ If I want to test myself in the browser, even before I launch the test, I will be able to do so using a css selector, which is not the case with an xpath selector. For example `document.querySelectorAll('.popper > div > div > div')` would do the job right from the console.